src/GitVersion.Core/Configuration/Init/BuildServer/AppVeyorSetup.cs

```diff
@@ -5,12 +5,6 @@
 
 namespace GitVersion.Configuration.Init.BuildServer;
 
-internal enum ProjectVisibility
-{
-    Public = 0,
-    Private = 1
-}
-
 internal class AppVeyorSetup : ConfigInitWizardStep
```

Comment:
FYI, I don't think `AppVeyorSetup` belongs in `GitVersion.Core`. The entire `Init` directory should probably be moved into the `GitVersion.App` project. Not something we need to deal with in this PR, but perhaps in a future one.

Reply:
I agree with that, and I wonder if we can deprecate the 'init' in v6 as it's not maintained and it does not cover all the options we currently have implemented. What do you think @asbjornu

Reply:
I agree that if we are to keep it, it needs a major revamp. So let's say that if no one invests time to revamp it completely until the release of `6.0.0`, we'll scrap it entirely.

Reply:
I created #3229 to track this.